packages/suite/src/actions/suite/analyticsActions.ts

```diff
@@ -63,6 +63,7 @@ export type AnalyticsEvent =
               theme: string;
               // added in 1.6
               suiteVersion: string;
+              desktopOSVersion: { platform: string; release: string } | '';
```

Comment:
not sure here. how does it look like serialized? If I remember correctly, properties on `payload` object were always primitives till now.

Reply:
Yes, all properties in the payload are primitive till now. Maybe sending string in a format `${platform}_${release}` would be better?